azurerm/internal/services/lighthouse/lighthouse_definition_resource.go

```diff
@@ -73,6 +73,12 @@ func resourceLighthouseDefinition() *schema.Resource {
 							Required:     true,
 							ValidateFunc: validation.IsUUID,
 						},
+
+						"principal_id_display_name": {
```

Comment:
I don't think ID makes sense in the name here?
```suggestion
						"principal_display_name": {
```